src/components/main.js

```diff
@@ -29,9 +29,34 @@ export default function Header() {
     setFile(event.target.files[0]);
   };
 
+  const barGraphResults = results => {
+    return results.map((res, index) => {
+      let resToHeight = Math.round((res * 100) / 4) * 4;
+      // To display something at least
+      if (resToHeight === 0) {
```

Comment:
might be able to just use double equals since we know the types match.

Reply:
Yea, I normally just go w === tho. Not sure what is better stylistically, since both are equiv here